docs/reference/src/language/builtins/elements.md

```diff
@@ -138,6 +138,30 @@ The maximum and preferred size of the `Flickable` are based on the viewport.
 When not part of a layout, its width or height defaults to 100% of the parent
 element when not specified.
 
+### Pointer Event Interaction
+
+If the `Flickable`'s area contains elements that use `TouchArea` to act on clicking, such as `Button`
+widgets, then the following algorithm is used to distinguish between the user's intent of scrolling or
+interacting with `TouchArea` elements:
+
+1. If the `Flickable`'s `interactive` property is `false`, all events are forwarded to elements underneath.
+2. Any wheel events are interpreted as attempt of the user to scroll.
+3. If a press event is received where the event's coordinates interact with a `TouchArea`, the event is stored
+   and any subsequent move and release events are handled as follows:
+   1. If 100ms elapse without any events, the stored press event is delivered to the `TouchArea`.
+   2. If a release event is received before 100ms have elapsed, the stored press event as well as the
+      release event are immediately delivered to the `TouchArea` and the algorithm resets.
+   3. Any move events received will start a flicking operation on the `Flickable` if all of the following
+      conditions are met:
+        1. The event is received before 500ms have elapsed since receiving the press event.
+        2. The euclidean distance to the coordinates of the press event exceeds 8 logical pixels.
+        3. The flickable has available free space in the direction of the move event.
```

Comment:
```suggestion
        2. The distance to the press event exceeds 8 logical pixels in an orientation in which we are allowed to move
```

It's not the euclidean distance.
and we start the flick even if we move down when there is no down space.  That's because otherwise the user knows that it can flick down and do it, but it reached the end so release it, and would press on things by mistake

Reply:
Oh, I misread. Thanks!